### PR TITLE
Fix code coverage analysis

### PR DESCRIPTION
**Pull request by gathogojr**
### Description

Fix code coverage analysis

Restrict code coverage analysis to only the ODL libraries:
{C2646442-5390-49A7-9FCA-C110E72E7C34}

### Checklist (Uncheck if it is not completed)

- [ ] *Build and test with one-click build and test script passed*

### CODE REVIEW

File: test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.csproj [hunk 1]
```diff
@@ -37,6 +37,10 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp1.1' or '$(TargetFramework)' == 'net452'">
```
Comment (xuzhg) [inline test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.csproj:40]: why do we still support 'netcoreapp1.1'?

File: test/FunctionalTests/Microsoft.OData.Core.Tests/Microsoft.OData.Core.Tests.csproj [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp1.1'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp1.1' or '$(TargetFramework)' == 'net452'">
```
Comment (xuzhg) [inline test/FunctionalTests/Microsoft.OData.Core.Tests/Microsoft.OData.Core.Tests.csproj:31]: Shall we remove 'netcoreapp1.1'?

**Review by xuzhg — APPROVED** [review]

File: test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.csproj (hunk 1 above)
Comment (habbes) [reply test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.csproj:40]: Seems that this PR is targeted at 7.x branch.

**Review by habbes — APPROVED** [review]

File: test/FunctionalTests/Microsoft.OData.Core.Tests/Microsoft.OData.Core.Tests.csproj (hunk 2 above)
Comment (gathogojr) [reply test/FunctionalTests/Microsoft.OData.Core.Tests/Microsoft.OData.Core.Tests.csproj:31]: This pull request is targeted at 7.x

File: .gitignore [hunk 3]
```diff
@@ -15,6 +15,7 @@ obj/
 
 # Test results
 sln/TestResults/
+**/TestResults/
```
Comment (marabooy) [inline .gitignore:18]:
```suggestion
**/TestResults/
```
nit

**Review by marabooy — DISMISSED** [review]

**Review by habbes — APPROVED** [review]

**Review by marabooy — APPROVED** [review]

**Review by habbes — APPROVED** [review]

**Pull request merged**